gr-fec/lib/async_decoder_impl.cc

```diff
@@ -112,88 +107,56 @@ void async_decoder_impl::decode_unpacked(pmt::pmt_t msg)
 
     size_t o0(0);
```

Comment:
I know it's been there. But could you rename this variable? The name bears no meaning, is easily confused with a literal. It looks a lot like someone read [this](https://github.com/Droogans/unmaintainable-code) or smth similar.

Reply:
I also didn't like this, but for some reason I decide not to rename variables too much. I've renamed this to `vector_len` and added a comment about what this variable is for, because the only reason we're using it is because `f32vector_elements` and the like require to output the vector length in a variable. We don't use the value.